### ISSUE TITLE
Demo: 'Official gov't website' banner - hover & arrow colors need to change

### ISSUE THREAD

**Issue by ToniBonittoGSA** [GSA/digitalgov-pra#227]
"Here's how you know " -- has dark blue hover text and dark blue down arrow on a black background; some folks will have a very hard time seeing it.

**Comment by jeremyzilar** [comment]
I will have to fix this.

**Comment by cmajel** [comment]
@jeremyzilar as a reference, here's what I had for proposed styling in the new header:
*default*
![default](https://user-images.githubusercontent.com/2374206/59811289-2f422a00-92d7-11e9-8e03-1bcf630fac2b.jpg)

*expanded*
![gov-site-expanded](https://user-images.githubusercontent.com/2374206/59811290-2f422a00-92d7-11e9-9004-daf363303e2a.jpg)

**Comment by jeremyzilar** [comment]
That's fairly easy to make happen @cmajel 🎉
